### ISSUE TITLE
[Feature Request] Parsing Old Kana Orthography (歴史的仮名遣い) to Modern Kana Orthography (現代仮名遣い)

### ISSUE THREAD

**Issue by tabinogusha** [FooSoft/yomichan#2285]
To may whom it concern:

Good day.

I would like to suggest an optional feature that enables one to parse words that are written in old orthography and automatically convert it into its modern respective variants. For example: 言ふ into 言う, だつた into だった, and でせう into でしょう ad nauseam. One can always, of course, manually convert the words into its modern orthography by using the Yomichan search feature, which in the end, destroys the purpose of pop-up dictionaries which is to look up words in the matter of seconds. Therefore, automatic conversion from old orthography into modern othography relieves one of the burden of manual conversion, thus enabling them to see dictionary entries and add words into Anki instantaneously just like what they would normally do in any reading activity. In addition, creating new dictionaries in their old orthography would also become unnecessary as this feature would allow one to use working dictionaries without any changes at all. 

Thank you for reading and I'm looking forward for your response.

**Comment by Aquafina-water-bottle** [comment]
I'm not a developer of Yomichan, but this is a related question to the developers: how feasible/reasonable would it be to allow users to provide their own `deinflect.json`, say in the Yomichan options menu? There seems to be a decent chunk of issues / pull requests relating to the deinflector options (including this one), so I feel like having it be customizable would be a good general solution to many of those.

As for me personally, I personally want to remove `vs` from `imperative negative` since it can occasionally separate the JMdict entries from the monolingual dictionaries (this is especially common when a word is used as a na-adj instead of a suru verb). However, that's not something I want to impose on all other users, and adding an option for that in particular seems too specific for the problem, so I believe a customizable deflector would be the best option in that case.

**Comment by tabinogusha** [comment]
I've simply made an account to submit a suggestion; I have no knowledge in regards to coding or computer science. What is a deinflect.json in layman's terms?

**Comment by Aquafina-water-bottle** [comment]
It's just a file that defines the inflections that allow conjugated words to be properly parsed by Yomichan. You can see the file here, it should be more or less self explanatory: https://github.com/FooSoft/yomichan/blob/master/ext/data/deinflect.json
